packages/react-app/src/components/BuilderRow.jsx

```diff
@@ -13,7 +13,7 @@ import useCustomColorModes from "../hooks/useCustomColorModes";
 
 const secondsPerDay = 24 * 60 * 60;
 
-const BuilderRow = ({ builder, mainnetProvider }) => {
+const BuilderRow = (({ builder, mainnetProvider }) => {
```

Comment:
Things like this should be fixed by the linter or prettier. @simplemachine92 please make sure you have them configured.

I'll fix the ones I find on this PR because I'm making changes here as well, so don't worry about this PR. Just mentioning for the future.